utils/common/src/main/kotlin/Utils.kt

```diff
@@ -26,13 +26,14 @@ import java.util.EnumSet
 
 /**
  * A list of directories used by version control systems to store metadata.
+ * The list covers also version control systems not supported by ORT, because such directories
```

Comment:
Commit title: I guess a more fitting type would be "docs".

Reply:
Does this deliberately start in a new line instead of continuing the first line of the comment? If so, why?